src/main/webapp/js/userMap.js

```diff
@@ -14,20 +14,25 @@ function handleData(err, countryCoordinates, userData) {
         handleError();
         return;
     }
-    var userCountries = Object.keys(userData);
     var countriesArr = [];
     var total = 0;
-    userCountries.forEach(function(countryName) {
-        var countryTotal = userData[countryName].length;
-
-        countriesArr.push([countryName, countryTotal]);
-        total += countryTotal;
-    });
-
+    var countryTotal = 0;
+    var date = userData.lastUpdated;
+    for (key in userData.institutes) {
```

Comment:
@wkurniawan07 does it really needed to use `Object.keys()` ?. I think its a waste of time simply calling that method as the number of objects are 2 unlike the previous case.